ospo-mindmap/Content/ospomindmap.md

```diff
@@ -148,6 +149,7 @@ development team of the Linux Project
 - Policy on making open source business decisions
   - Risk assessment and acceptance  
   - Business alignment
+- Help navigate internal politics or policies
```

Comment:
"Help navigate internal politics or policies" is not only a compliance topic but also a contributing to Open Source  - what if an engineer/product owner wants to open-source a big closed source application with several internal dependencies and approvals from multiple directories and above. OSPO is the team that can help with these kind of tasks that require existing relationships to higher management